vitrina/datasets/models.py

```diff
@@ -128,6 +128,27 @@ def get_absolute_url(self):
     def get_tag_list(self):
         return str(self.tags).replace(" ", "").split(',') if self.tags else []
 
+    def get_members_url(self):
+        return reverse('dataset-members', kwargs={'pk': self.pk})
+
+
+class DatasetMember(models.Model):
```

Comment:
Please see #171 task, dataset representatives should be implemented using generic `Representative` model, please consult and collaborate with the #171 task author.